dtmi/sgs/stas_plus-1.json

```diff
@@ -0,0 +1,68 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:sgs:STAS_PLUS;1",
+  "@type": "Interface",
+  "displayName": "STAS_PLUS",
+  "description": "Report the current oil quality data and oil iron content distribution data, and Get Log for STAS-PLUS.",
+  "contents": [
+    {
+      "@type": [
+        "Property",
+        "SP_Name"
+      ],
+      "name": "ModelNum",
+      "displayName": "PLUS Model",
+      "description": "Identification model",
+      "schema": "string"
+    },
+    {
+      "@type": [
+        "Property",
+        "SP_Temper"
+      ],
+      "name": "SPTemperatureT1",
+      "displayName": "thermography",
+      "description": "Record temperature channel 1",
```

Comment:
It is highly recommended to follow the DTDL naming guide here https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines#interfaces, which describes best practices for public models. It lists recommendations such as descriptions should end in proper punctuation (like with period '.' here) and display name should start with a capital letter.

Also here is a section for element name best practice https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines#element-names.